workflow/workflow-demos/processes/BusinessNotification/OrderProcess.p.json

```diff
@@ -0,0 +1,385 @@
+{
+  "$schema" : "https://json-schema.axonivy.com/process/11.2.2/process.json",
+  "id" : "18DD058BCE48D418",
+  "config" : {
+    "data" : "workflow.order.OrderData"
+  },
+  "elements" : [ {

```

Comment:
I'd add an 'annotation' on this process, explaining the purpose of this demo. ... as you see e.g. in procurementRequest